go/vt/vtctl/workflow/utils.go

```diff
@@ -850,6 +850,18 @@ func getTenantClause(vrOptions *vtctldatapb.WorkflowOptions,
 	return &sel.Where.Expr, nil
 }
 
+func changeKeyspaceRoute(ctx context.Context, ts *topo.Server, tabletTypes []topodatapb.TabletType, sourceKeyspace string, routedKeyspace string) error {
```

Comment:
SUPER nitty (sorry), but `changeKeyspaceRouting` might be a better name with better param names being `fromKeyspace, toKeyspace string`